docs/src/literate-tutorials/linear_elasticity.jl

```diff
@@ -1,3 +1,408 @@
+# facets in the relevant facetset. We do this by using the [`FacetIterator`](@ref).
+
+function assemble_external_forces!(f_ext, dh, facetset, facetvalues, prescribed_traction)
+    ## Create a temporary array for the facet's local contributions to the external force vector
+    fe_ext = zeros(getnbasefunctions(facetvalues))
+    for face in FacetIterator(dh, facetset)
+        ## Update the facetvalues to the correct facet number
+        reinit!(facetvalues, face)
+        ## Reset the temporary array for the next facet
+        fill!(fe_ext, 0.0)
+        ## Access the cell's coordinates
+        cell_coordinates = getcoordinates(face)
+        for qp in 1:getnquadpoints(facetvalues)
+            ## Calculate the global coordinate of the quadrature point.
+            x = spatial_coordinate(facetvalues, qp, cell_coordinates)
+            tₚ = prescribed_traction(x)
+            ## Get the integration weight for the current quadrature point.
+            dΓ = getdetJdV(facetvalues, qp)
+            for i in 1:getnbasefunctions(facetvalues)
+                Nᵢ = shape_value(facetvalues, qp, i)
+                fe_ext[i] += tₚ ⋅ Nᵢ * dΓ
+            end
+        end
+        ## Add the local contributions to the correct indices in the global external force vector
+        assemble!(f_ext, celldofs(face), fe_ext)
+    end
+    return f_ext
+end
+#md nothing #hide
+
+# ### Material behavior
+# Next, we need to define the material behavior, specifically the elasticity tensor $\mathsf{C}$.
+# In this tutorial, we use plane strain linear isotropic elasticity, with Hooke's law as
+# ```math
+# \boldsymbol{\sigma} = 2G \boldsymbol{\varepsilon}^\mathrm{dev} + 3K \boldsymbol{\varepsilon}^\mathrm{vol}
+# ```
+# where $G$ is the shear modulus and $K$ the bulk modulus. The linearity of this expression implies that
+# can be represented by the general expression, $\boldsymbol{\sigma} = \mathsf{C}:\boldsymbol{\varepsilon}$,
+# from above, with
+# ```math
+#  \mathsf{C} := \frac{\partial \boldsymbol{\sigma}}{\partial \boldsymbol{\varepsilon}}
+# ```
+# The volumetric, $\boldsymbol{\varepsilon}^\mathrm{vol}$,
+# and deviatoric, $\boldsymbol{\varepsilon}^\mathrm{dev}$ strains, are defined as
+# ```math
+# \begin{align*}
+# \boldsymbol{\varepsilon}^\mathrm{vol} &= \frac{\mathrm{tr}(\boldsymbol{\varepsilon})}{3}\boldsymbol{I}, \quad
+# \boldsymbol{\varepsilon}^\mathrm{dev} &= \boldsymbol{\varepsilon} - \boldsymbol{\varepsilon}^\mathrm{vol}
+# \end{align*}
+# ```
+#
+# Starting from Young's modulus, $E$, and Poisson's ratio, $\nu$, the shear and bulk modulus are
+# ```math
+# G = \frac{E}{2(1 + \nu)}, \quad K = \frac{E}{3(1 - 2\nu)}
+# ```
+Emod = 200e3 # Young's modulus [MPa]
+ν = 0.3      # Poisson's ratio [-]
+
+Gmod = Emod / (2(1 + ν))  # Shear modulus
+Kmod = Emod / (3(1 - 2ν)) # Bulk modulus
+
+# Finally, we demonstrate `Tensors.jl`'s automatic differentiation capabilities when
+# calculating the stiffness tensor
```

Comment:
I think it is helpful to stick to a nomenclature, once introduced, and avoid synonyms, to not throw off readers without consolidated knowledge about linear elasticity (which might jump on the train of thought: Why is this now called differently? Is it a different object?).
```suggestion
# calculating the elasticity tensor
```

Reply:
Good catch! Now it says "elastic stiffness tensor" everywhere to be clear.